js/main.js

```diff
@@ -25,4 +26,90 @@ function handleSubmission(e) {
   data.entries.unshift(newEntry);
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entry.reset();
+
+  const submission = renderEntry(newEntry);
+  $entryList.prepend(submission);
+  viewSwap('entries');
+  toggleNoEntries();
+}
+
+function renderEntry(entry) {
+  const $liRow = document.createElement('li');
+  $liRow.setAttribute('class', 'row');
+
+  const $imgDiv = document.createElement('div');
+  $imgDiv.setAttribute('class', 'column-half');
+  $liRow.appendChild($imgDiv);
+
+  const $img = document.createElement('img');
+  $img.setAttribute('src', entry.image);
+  $img.setAttribute('alt', entry.title);
+  $imgDiv.appendChild($img);
+
+  const $noteText = document.createElement('div');
+  $noteText.setAttribute('class', 'column-half');
+  $liRow.appendChild($noteText);
+
+  const $noteTitle = document.createElement('h3');
+  $noteTitle.textContent = entry.title;
+  $noteText.appendChild($noteTitle);
+
+  const $noteContent = document.createElement('p');
+  $noteContent.textContent = entry.notes;
+  $noteText.appendChild($noteContent);
+
+  return $liRow;
+}
+
+document.addEventListener('DOMContentLoaded', handleDOMContentLoaded);
+
+function handleDOMContentLoaded(e) {
+  for (let i = 0; i < data.entries.length; i++) {
+    viewSwap(data.view);
```

Comment:
This should be getting called outside your `for` loop. Otherwise you're calling this function for every item in the array.